Dockerfile

```diff
@@ -24,34 +24,30 @@
 # Dockerfile for Cray Boot Orchestration Service (BOS)
 
 # Upstream Build Args
-ARG OPENAPI_IMAGE=openapitools/openapi-generator-cli:v4.1.2
-ARG ALPINE_BASE_IMAGE=artifactory.algol60.net/docker.io/alpine:3.14
+ARG OPENAPI_IMAGE=openapitools/openapi-generator-cli:v6.0.1
+ARG ALPINE_BASE_IMAGE=artifactory.algol60.net/csm-docker/stable/docker.io/library/alpine:3
 
 # Generate Code
 FROM $OPENAPI_IMAGE as codegen
 WORKDIR /app
 COPY api/openapi.yaml api/openapi.yaml
 COPY config/autogen-server.json config/autogen-server.json
-COPY config/autogen-client.json config/autogen-client.json
 RUN /usr/local/bin/docker-entrypoint.sh generate \
     -i api/openapi.yaml \
     -g python-flask \
     -o lib \
     -c config/autogen-server.json \
     --generate-alias-as-model
-RUN /usr/local/bin/docker-entrypoint.sh generate \
```

Comment:
Did you delete this and the previous lines because we weren't using them?

Reply:
Yes; as it happens it was beginning to look impossible to satisfy both client and server code. Since the client code wasn't being used, I disabled generating it.